WPMedia/fusion-news-theme-blocks
# WIP: TMEDIA-36) Focus correction for search-results-list

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
_Information about what you changed for this PR_

## Jira Ticket
- [TMEDIA-36](https://arcpublishing.atlassian.net/browse/TMEDIA-36)

## Acceptance Criteria
When clicking the 'See More' button, the results load above the button and the focus remains on the last story rather than the first of the newly loading stories. 

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch git checkout TMEDIA-36/see_more_focus_correction
2. In Fusion repo run with share block linked npx fusion start -f -l @wpmedia/search-results-list-block
3. Navigate to the follow page - http://localhost/search/lorem-ipsum/?_website=the-gazette
4. Scroll down to see the "See More" button
5. Click on the button
6. After results have loaded click the tab button on your keyboard
7. Next item to be focused is the 11th article - the first one from the next set to be added

## Effect Of Changes
### Before
'See More' results load above and the focus remains on the last story of the results.

### After
'See More' results focus should be on the first of the loaded stories.

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
